### DISCUSSION TITLE
tutorial formatting issue

### DISCUSSION THREAD

**Discussion by elizabethkeith** [IDAES/idaes-pse#1430] [General]
Hi there, I'm working through the tutorials and "HDA Flowsheet Simulation and Optimization" tutorial page looks like this:

<img width="1434" alt="Screenshot 2024-06-10 at 2 15 08 PM" src="https://github.com/IDAES/idaes-pse/assets/113551905/b318aa13-2f7e-47d5-94c4-d9eb14e101b0">

It's really difficult to work through since the formatting is so wonky so I thought I'd draw someone's attention to it, if no one else already has. Thanks!

**Comment by andrewlee94** [comment]
@elizabethkeith Thank you for bringing this to our attention - we noticed this ourselves last week and are trying to work out what is going wrong.

In the mean time, you can view the source notebooks directly at https://github.com/IDAES/examples/tree/main/idaes_examples/notebooks/docs/tut
